internal/circonus/metric_filters.go

```diff
@@ -85,10 +66,103 @@ func (c *Check) defaultFilters() [][]string {
     ["allow", "^events$", "events"],
     ["allow", "^collect_.*$", "agent collection stats"],
     ["allow", "^authentication_attempts$", "api auth health"],
+    ["allow", "^cadvisor.*$", "cadvisor"],
     ["deny", "^.+$", "all other metrics"]
     ]
 }
-`)
+`
+	defaultMetricFiltersStr118 = `
+{
+    "metric_filters": [
+	["allow", "^.+$", "tags", "and(collector:dynamic)", "NO_LOCAL_FILTER dynamically collected metrics"],
+    ["allow", "^(pod|node)_cpu_usage_seconds_total$", "utilization"],
+    ["allow", "^(pod|node)_memory_working_set_bytes$", utilization"],
+    ["allow", "^(kube_)?pod_container_status_(running|terminated|waiting|ready)(_count)?$", "containers"],
+    ["allow", "^(kube_)?pod_container_status_(terminated|waiting)_reason(_count)?$", "containers health"],
+    ["allow", "^(kube_)?pod_init_container_status_(terminated|waiting)_reason(_count)?$", "init containers health"],
+    ["allow", "^kube_deployment_(created|spec_replicas)$", "deployments"],
+    ["allow", "^kube_job_status_failed$", "health"],
+    ["allow", "^kube_persistentvolume_status_phase$", "health"],
+	["allow", "^kube_deployment_status_replicas_unavailable$", "deployments"],
```

Comment:
This filter should be removed.

Reply:
resolved in [d71873a](https://github.com/circonus-labs/circonus-kubernetes-agent/pull/91/commits/d71873af18c6bbca5d8fb9c2066daf06d888b9f6)